MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include atool/templates *.html
```

Comment:
> If you don’t supply an explicit list of files (or instructions on how to generate one), the sdist command puts a minimal default set into the source distribution:
> [...]
>  - all files that matches the `package_data` metadata. See Installing Package Data.

Does it not? https://docs.python.org/2.7/distutils/sourcedist.html#specifying-the-files-to-distribute